### DISCUSSION TITLE
Project has launched????

### DISCUSSION THREAD

**Discussion by Aryaajain119** [github/india#516] [General]
Anyone here whose project is launched????
Mine it's still showing the project will be launched on 30th October

**Comment by ShivaniPawade** [comment]
yes  the same issue i am also facing

**Reply by Gdeekonda** [reply]
yes i am also facing same issues ane i have completed every modules but it is still in process

**Comment by RajyaLakshmiAnisetti** [comment]
yes, I am also facing same problem

**Comment by rishabh-vasudevan** [comment]
It shows the same for me
